app/models/spree/line_item_decorator.rb

```diff
@@ -56,6 +57,10 @@ def included_tax
     adjustments.included_tax.sum(&:included_tax)
   end
 
+  def tax_rates
+    product.tax_category.tax_rates
```

Comment:
Would be nice to have some sort of documentation explaining this, something like the following:
```ruby
# @return [Array<String>]
def tax_rates
  return [] unless product.tax_category

  product.tax_category.tax_rates
end
```